tasks/smoketests/vscode/quick_open.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import time
+from platform import platform
+from typing import List
+from dataclasses import dataclass
+from enum import Enum
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from .base import QuickOpen as BaseQuickOpen, Component
+from ..utils.tools import Platform, get_platform
+
+QUICK_OPEN = "div.monaco-quick-open-widget"
+QUICK_OPEN_HIDDEN = 'div.monaco-quick-open-widget[aria-hidden="true"]'
+QUICK_OPEN_INPUT = f"{QUICK_OPEN} .quick-open-input input"
+QUICK_OPEN_FOCUSED_ELEMENT = (
+    f"{QUICK_OPEN} .quick-open-tree .monaco-tree-row.focused .monaco-highlighted-label"
+)
+QUICK_OPEN_ENTRY_SELECTOR = 'div[aria-label="Quick Picker"] .monaco-tree-rows.show-twisties .monaco-tree-row .quick-open-entry'
+QUICK_OPEN_ENTRY_LABEL_SELECTOR = 'div[aria-label="Quick Picker"] .monaco-tree-rows.show-twisties .monaco-tree-row .quick-open-entry .label-name'
+
+
+class QuickOpen(BaseQuickOpen, Component):
+    def select_command(self, command: str, **kwargs):
+        ele = self._open(command, **kwargs)
+        self.app.core.dispatch_keys(Keys.ENTER, element=ele)
+
+    def select_value(self, value: str, **kwargs):
+        ele = self._wait_until_opened(5)
+        self.app.core.dispatch_keys(value, element=ele)
+        self.app.core.dispatch_keys(Keys.ENTER, element=ele)
+
+    def wait_until_selected(self, value: str, **kwargs):
+        def find(eles):
+            if len(eles) > 0:
```

Comment:
```suggestion
            if eles:
```
(I'm assuming `eles` is a sequence type.)